src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -181,6 +182,15 @@ def get_schema_value_str(record)
         return resultStr
     end
 
+    # Convert (unify) mdsd.syslog.** tags to mdsd.syslog so that mdsd sees only
+    # one single tag for all syslog messages. This is the use case for basic
+    # syslog messages collection. Also, it appears that tags can't be changed
+    # by a fluentd filter, so they need to be changed here in this output plugin.
+    def create_mdsd_source(tag)
+        if tag.start_with?("mdsd.syslog.")
```

Comment:
Thanks @LiweiPeng . This makes sense. I was just following the OMS filter_syslog example. Pushed another commit reflecting your suggestion. Please review again.